### ISSUE TITLE
is there a way to select multiple items through code?

### ISSUE THREAD

**Issue by gietal** [jwilling/JNWCollectionView#184]
found this on JNWCollectionViewFramework.m:
(void)selectItemsAtIndexPaths:(NSArray *)indexPaths animated:(BOOL)animated

but it seems like this function is private, what's the reason for this? it seems like a useful code

**Comment by jwilling** [comment]
I believe this wasn't exposed because it was very simple to reproduce outside of the framework by looping over selected indexes. Do you think it's valuable enough to be public?

**Comment by gietal** [comment]
the only public selection function I see is 
(void)selectItemAtIndexPath:(NSIndexPath *)indexPath atScrollPosition:(JNWCollectionViewScrollPosition)scrollPosition animated:(BOOL)animated

which defaults to single selection and will deselect my previously selected items

I would at least need this function to be public, if I want to loop through my indexes and select them one by one:
(void)selectItemAtIndexPath:(NSIndexPath *)indexPath animated:(BOOL)animated

**Comment by jwilling** [comment]
You're right, I apologize. It's been a very long time since I've looked at this code. If you want to make a pull request to make this public that'd be great!
